src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/JsonSchemaEntity.kt

```diff
@@ -53,6 +53,16 @@ class Cas2ApplicationJsonSchemaEntity(
   schema: String,
 ) : JsonSchemaEntity(id, addedAt, schema)
 
+@Entity
+@DiscriminatorValue("CAS_2_BAIL_APPLICATION")
+@Table(name = "cas_2_bail_application_json_schemas")
+@PrimaryKeyJoinColumn(name = "json_schema_id")
+class Cas2BailApplicationJsonSchemaEntity(
+  id: UUID,
+  addedAt: OffsetDateTime,
+  schema: String,
+) : JsonSchemaEntity(id, addedAt, schema)
```

Comment:
i'm not sure if you want to do this now, but we've decided to remove all the Json Schema/Json Validation code and tables. See https://dsdmoj.atlassian.net/browse/APS-1570 for what we think can be removed.